src/Makefile

```diff
@@ -1500,90 +1507,21 @@ EXTRA_CFLAGS += -fPIC
 RTOBJS := $(sort $(foreach mod,$(obj-m),$(call TORTOBJS,$(mod))))
 
 RTDEPS := $(sort $(patsubst $(OBJDIR)/%.o,$(DEPDIR)/%.d, $(RTOBJS)))
-
```

Comment:
Honestly I have no idea and was wondering this myself.
I was trying to see If there had been any other changes to that makefile that has crossed my
longtime PR that has been reverted and rebased attempted (by me)... and messed up